BizHawk.Client.Common/SharpCompressArchiveHandler.cs

```diff
@@ -44,7 +44,8 @@ public bool CheckSignature(string fileName, out int offset, out bool isExecutabl
 			catch { }
 			finally
 			{
-				arcTest.Dispose();
+				if (arcTest != null)
```

Comment:
Pretty sure this can be `arcTest?.Dispose();`. Other than that, I dunno, if it works it works.